src/react-jw-player.jsx

```diff
@@ -82,6 +82,11 @@ class ReactJWPlayer extends Component {
 
     const component = this;
     const player = window.jwplayer(this.videoRef);
+    if (!player) {
```

Comment:
This won't actually work as that'll return the following:

`window.jwplayer("random")` => `{registerPlugin: ƒ}`